### PR TITLE
Merge 'apple' branch with 'master' branch.

### PR DESCRIPTION
**Pull request by paulgureghian**

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job with the project @paulgureghian :+1: 
However, the project is still far from being merged. There are a lot of changes to be made. Please look at my suggestions below;
- [ ] add a navbar with 9 buttons. the navbar should be implemented with ```background -> gradient ```
- [ ] include a footer wrapped in ```<footer>``` tag
- [ ] your page should also include four elements with links
- [ ] make your clone as close to the original page as possible. Check the original site [here](https://web.archive.org/web/20140301004610/http://www.apple.com/)

### Please make those changes and re-submit for another review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Apple</title>
+  <link href="styles.css type=" text/css" rel="stylesheet">
```
Comment (Redvanisation) [inline index.html]: You have an error on here, please link the CSS file properly

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @paulgureghian 🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. Your CSS file is empty, please implement and apply the necessary styles.

2. The `<link>` tag has an error, see my comment below and fix it.

3. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

4. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by stratospherique — CHANGES_REQUESTED** [review]
> #### Pull request under review by Ahmed Mahfoudh (TSE)

### Hi @paulgureghian 

- I still cannot find the content of your _styles.css_ file and the live preview also shows an un-styled webpage.

- **[REQUIRED]** refer to the comments of the reviewer above :arrow_up: . They should guide you through getting your project approved.

### that's it :relieved: . When finished, kindly resubmit another code review request :pray:

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @paulgureghian 

### _Well done correcting the HTML markup, however you still have some issues that need to be fixed before I can approve the project:_ 

1. Your CSS file has only one class, please fill-up the styles needed for this project and push it to your repo before submitting a new code review request, this is how the website is looking on my screen (it needs to be looking exactly like the original website): 
Screen Shot 2019-10-09 at 4 36 14 PM


2. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

- If you need help with that you can reach out to me on slack: **_@redvan_**

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @paulgureghian 

**Good work so far but the project is still not looking like the original website, please refer to the page original page in [here](https://web.archive.org/web/20140301004610/http://www.apple.com/) and try to style your project according to it.**

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by ebukaume** [comment]
# Changes required! :stop_sign: 
Hi @paulgureghian :wave: 

Please make the requested changes. You can start by first making the image a background image. You can refer to [this article](https://css-tricks.com/almanac/properties/b/background-image/). 

Then read about how to use flexbox [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/). With these, you can achieve most of the desired result.

**Kindly submit another code review when your page looks like the one [here](https://web.archive.org/web/20140301004610/http://www.apple.com/) and there are no lint errors**


Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Comment by paulgureghian** [comment]
Can I ask you a question here ?

**Comment by ebukaume** [comment]
> Can I ask you a question here ?

Yes, please.

**Comment by paulgureghian** [comment]
<nav id="globalheader" class="apple">
    <ul id="globalnav" role="navigation">
        <li id="gn-apple"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/"><span>Apple</span></a></li>
        <li id="gn-store"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"><span>Store</span></a></li>
        <li id="gn-mac"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/"><span>Mac</span></a></li>
        <li id="gn-ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"><span>iPod</span></a></li>
        <li id="gn-iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"><span>iPhone</span></a></li>
        <li id="gn-ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"><span>iPad</span></a></li>
        <li id="gn-itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"><span>iTunes</span></a></li>
        <li id="gn-support" class="gn-last"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"><span>Support</span></a></li>

**Comment by paulgureghian** [comment]
<nav id="globalheader" class="apple">
    <ul id="globalnav" role="navigation">
        <li id="gn-apple"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/"><span>Apple</span></a></li>
        <li id="gn-store"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"><span>Store</span></a></li>
        <li id="gn-mac"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/"><span>Mac</span></a></li>
        <li id="gn-ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"><span>iPod</span></a></li>
        <li id="gn-iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"><span>iPhone</span></a></li>
        <li id="gn-ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"><span>iPad</span></a></li>
        <li id="gn-itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"><span>iTunes</span></a></li>
        <li id="gn-support" class="gn-last"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"><span>Support</span></a></li>

**Comment by paulgureghian** [comment]
`            <nav id="globalheader" class="apple">
    <ul id="globalnav" role="navigation">
        <li id="gn-apple"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/"><span>Apple</span></a></li>
        <li id="gn-store"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"><span>Store</span></a></li>
        <li id="gn-mac"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/"><span>Mac</span></a></li>
        <li id="gn-ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"><span>iPod</span></a></li>
        <li id="gn-iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"><span>iPhone</span></a></li>
        <li id="gn-ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"><span>iPad</span></a></li>
        <li id="gn-itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"><span>iTunes</span></a></li>
        <li id="gn-support" class="gn-last"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"><span>Support</span></a></li>
                                 `

**Comment by paulgureghian** [comment]
Shouldnt the text in the span tags actually render on the navbar buttons ?

**Comment by ebukaume** [comment]
> Shouldnt the text in the span tags actually render on the navbar buttons ?

They should and they do. The problem is with the styles you applied. Have you considered using button tags? It would be much simpler `<button>Button Name</button>`.

**Comment by suretrust** [comment]
**Great job thus far @paulgureghian. :+1:** 

The clone doesn't look much like [this](https://web.archive.org/web/20140301004610/http://www.apple.com/)

Please take your time and go at your pace, look at the page you want to clone. Inspect the page using developer tools if necessary, then go ahead to start cloning. You can read [this article](https://www.lifewire.com/web-browser-developer-tools-3988965) to know how to use developer tools.

Also, be sure to have no linter error when submitting another code review. 

_Overall, I love your tenacity, keep up the great work!_

**Kindly re-submit for another code review once you have fixed these issues. Good luck! :tada:**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		  <ul>
+			<li id="apple"><a href="/web/20140301004610/http://www.apple.com/"></a></li>                
+			<li id="store"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"></a></li>
+			<li id="mac"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/"></a></li>
+			<li id="ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"></a></li>
+			<li id="iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"></a></li>
+			<li id="ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"></a></li>
+			<li id="itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"></a></li>
+			<li id="support"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"></a></li>
+			<li id="search"></li>
```
Comment (Redvanisation) [inline index.html]: The search bar should be an `<input type="text" class="search">` then style the class search in the CSS file,  and not a CSS image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		  <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/">
+			 <img src="images/your_verse_title_small.png" alt="title"/>
+		  </a>
+	  
+		  <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/">
+			 <img src="images/your_verse_subhead_small.png" alt="subhead"/>
+		  </a>	
+		
+		  <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/#video-your-verse/">
+			 <img src="images/cta_your_verse_27pt.png" alt="cta"/>
```
Comment (Redvanisation) [inline index.html:40]: The text is duplicated here please fix that

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+}
+
+.hero {
+  height: 538px;
+  position: relative;
+  background: url(images/your_verse_hero.jpg) 50% 100% no-repeat;
+  margin-bottom: 2px;
+}
+
+div {
```
Comment (Redvanisation) [inline styles.css]: Delete the whitespaces after the `div {`

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+  min-width: 980px;
+  width: 100%;
+  height: 90px;
+  padding: 0;
+  color: #6e6e6e;
+  font-size: 0.8234em;
+  line-height: 1.2;
+  text-align: center;
+  zoom: 1;
+  margin-top: 3px;
```
Comment (Redvanisation) [inline styles.css]: Delete the whitespaces after the `3px;`

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

**Excellent work so far, but we need to fix the issues below before moving on ⬇️** 

1. Please fix the search bar in the nav, I left a comment on how to do so below.

2. Delete the duplication in the main text, see my comment below about that.

3. Fix Stickler errors on the project, it has to be green before the project can be approved.

4. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by Anna-Myzukina** [comment]
#### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Paul!  
#### Wow, your project looks :+1: 
#### But need some Required Changes

### Page structure

#### Navbar

- [ ] It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section

*** Now your search field is under other buttons, but it should be on the rightmost section. Please fix this***
search

#### Bottom

* [ ] It contains four elements which are links (to another articles or `#`)

*** This elements should be links ***
links

Please delete h2 Promos inside section class="down" you don`t need it
*** With this element ***
before

*** Without this element ***
after


Your README.md file descriptive, but if you want you can use this short [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) to make this file more professional.

Great that you use [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links and the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer, but not bad to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document.


### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

File: index.html [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+			<li id="ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"></a></li>
+			<li id="iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"></a></li>
+			<li id="ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"></a></li>
+			<li id="itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"></a></li>
+			<li id="support"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"></a></li>
+		  </ul>
+		  
+		  <div id="search"> 
+			 <form action="https://web.archive.org/web/20140301004315/http://www.apple.com//search/" method="POST" class="search" id="g-search">
+
```
Comment (Anna-Myzukina) [inline index.html]: It should be on the rightmost section

File: index.html [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+		  </a>	
+		
+		  <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/#video-your-verse/">
+			 <img src="images/cta_your_verse_27pt.png" alt="cta"/>
+		  </a>
+	  </div>
+	</header>
+
+<section class="down">
+	<h2>Promos</h2>
```
Comment (Anna-Myzukina) [inline index.html]: Please delete <h2>Promos</h2> you don`t need this element

Comment (Anna-Myzukina) [inline index.html:46]: Please delete h2 Promos, you don`t need this element.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+		<img src="images/promo_iphone5c.jpg" alt="promo_iphone5c"/>
+	</div>
+
+    <div class="down-elem">
+		<img src="images/promo_30_years.jpg" alt="promo_30_years"/>
+    </div>
+
+    <div class="down-elem">
+		<img src="images/supplier.jpg" alt="promo_supplie_responsibility"/>
+	</div>
```
Comment (Anna-Myzukina) [inline index.html]: This elements should be links

File: index.html [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+			<li id="mac"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/"></a></li>
+			<li id="ipod"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/"></a></li>
+			<li id="iphone"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/"></a></li>
+			<li id="ipad"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/"></a></li>
+			<li id="itunes"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/"></a></li>
+			<li id="support"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/"></a></li>
+		  </ul>
+		  
+		  <div id="search"> 
+			 <form action="https://web.archive.org/web/20140301004315/http://www.apple.com//search/" method="POST" class="search" id="g-search">
```
Comment (Anna-Myzukina) [inline index.html]: It should be on the rightmost section

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project still needs changes

File: index.html [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
```
Comment (suretrust) [inline index.html]: Your navbar gradient design and placement still don't look close to the original website. Kindly fix.

**Yours**
nav
**Original**
nav2

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @paulgureghian,

### Amazing job thus far, :raised_hands: the images and background sections are lit! Just one more thing to go. Please see my comments on your code.

**Kindly submit for another code review once this has been fixed.** _Happy coding!_ :tada:

**Review by imhta — APPROVED** [review]
# Project Approved 👏
Hello @paulgureghian, Congratulations you have done great job. 






----
#### Don't forgot to sumbit project completion form
 **Pon Muthu Selvam N**
Feel free contact me 
[Github //@imhta](https://github.com/imhta) | [Twitter //@nponmuthuselvam](https://twitter.com/ponmuthuselvam)

**Pull request merged**